### DISCUSSION TITLE
老版本的简化首页找不到

### DISCUSSION THREAD

**Discussion by 2217936322** [the1812/Bilibili-Evolved#2471] [提问 / Q&A] [answered]
老版本的 简化首页找不到额 现在的简化首页的合集 不太友好  旧版本的使用习惯了

**Comment by the1812** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/26504152/140030564-df8edd29-6c1f-4473-b8e8-1056a5b5c279.png)
